### DISCUSSION TITLE
mumps on windows

### DISCUSSION THREAD

**Discussion by 1lihua** [scivision/mumps-superbuild#67] [Q&A] [answered]
Dear all
  
I am installing Mumps on my computer, but the following problem occurred. May I know the reason? I would greatly appreciate your suggestions.

-- MUMPS 5.6.1.2  CMake 3.25.3  Toolchain
-- Could NOT find MPI_Fortran (missing: MPI_Fortran_WORKS)
CMake Error at D:/cmakee/share/cmake-3.25/Modules/FindPackageHandleStandardArgs.cmake:230 (message):
  Could NOT find MPI (missing: MPI_Fortran_FOUND Fortran) (found version
  "2.0")
Call Stack (most recent call first):
  D:/cmakee/share/cmake-3.25/Modules/FindPackageHandleStandardArgs.cmake:600 (_FPHSA_FAILURE_MESSAGE)
  D:/cmakee/share/cmake-3.25/Modules/FindMPI.cmake:1837 (find_package_handle_standard_args)
  CMakeLists.txt:47 (find_package)

best wishes

**Comment by 1lihua** [comment]
There is some new troublie, when cmake --build build

[1/565] Linking Fortran static library mumps_common.lib
FAILED: mumps_common.lib
cmd.exe /C "cd . && C:\PROGRA~2\Intel\oneAPI\compiler\latest\windows\bin\ifx.exe /nologo CMakeFiles\mumps_common.dir\_deps\mumps-src\src\mumps_ooc_common.F.obj CMakeFiles\mumps_common.dir\_deps\mumps-src\src\mumps_static_mapping.F.obj CMakeFiles\mumps_common.dir\_deps\mumps-src\src\ana_omp_m.F.obj CMakeFiles\mumps_common.dir\_deps\mumps-src\src\double_linked_list.F.obj CMakeFiles\mumps_common.dir\_deps\mumps-src\src\fac_asm_build_sort_index_ELT_m.F.obj CMakeFiles\mumps_common.dir\_deps\mumps-src\src\fac_asm_build_sort_index_m.F.obj CMakeFiles\mumps_common.dir\_deps\mumps-src\src\fac_descband_data_m.F.obj CMakeFiles\mumps_common.dir\_deps\mumps-src\src\fac_future_niv2_mod.F.obj CMakeFiles\mumps_common.dir\_deps\mumps-src\src\fac_ibct_data_m.F.obj CMakeFiles\mumps_common.dir\_deps\mumps-src\src\fac_maprow_data_m.F.obj CMakeFiles\mumps_common.dir\_deps\mumps-src\src\front_data_mgt_m.F.obj CMakeFiles\mumps_common.dir\_deps\mumps-src\src\mumps_comm_ibcast.F.obj CMakeFiles\mumps_common.dir\_deps\mumps-src\src\mumps_l0_omp_m.F.obj CMakeFiles\mumps_common.dir\_deps\mumps-src\src\omp_tps_common_m.F.obj CMakeFiles\mumps_common.dir\_deps\mumps-src\src\ana_orderings_wrappers_m.F.obj CMakeFiles\mumps_common.dir\_deps\mumps-src\src\lr_common.F.obj CMakeFiles\mumps_common.dir\_deps\mumps-src\src\mumps_memory_mod.F.obj CMakeFiles\mumps_common.dir\_deps\mumps-src\src\mumps_mpitoomp_m.F.obj CMakeFiles\mumps_common.dir\_deps\mumps-src\src\ana_blk_m.F.obj CMakeFiles\mumps_common.dir\_deps\mumps-src\src\mumps_pivnul_mod.F.obj CMakeFiles\mumps_common.dir\_deps\mumps-src\src\mumps_addr.c.obj CMakeFiles\mumps_common.dir\_deps\mumps-src\src\mumps_common.c.obj CMakeFiles\mumps_common.dir\_deps\mumps-src\src\mumps_io_basic.c.obj CMakeFiles\mumps_common.dir\_deps\mumps-src\src\mumps_io_thread.c.obj CMakeFiles\mumps_common.dir\_deps\mumps-src\src\mumps_io_err.c.obj CMakeFiles\mumps_common.dir\_deps\mumps-src\src\mumps_io.c.obj CMakeFiles\mumps_common.dir\_deps\mumps-src\src\ana_orderings.F.obj CMakeFiles\mumps_common.dir\_deps\mumps-src\src\ana_set_ordering.F.obj CMakeFiles\mumps_common.dir\_deps\mumps-src\src\ana_AMDMF.F.obj CMakeFiles\mumps_common.dir\_deps\mumps-src\src\bcast_errors.F.obj CMakeFiles\mumps_common.dir\_deps\mumps-src\src\estim_flops.F.obj CMakeFiles\mumps_common.dir\_deps\mumps-src\src\mumps_type_size.F.obj CMakeFiles\mumps_common.dir\_deps\mumps-src\src\mumps_type2_blocking.F.obj CMakeFiles\mumps_common.dir\_deps\mumps-src\src\mumps_version.F.obj CMakeFiles\mumps_common.dir\_deps\mumps-src\src\mumps_print_defined.F.obj CMakeFiles\mumps_common.dir\_deps\mumps-src\src\mumps_numa.c.obj CMakeFiles\mumps_common.dir\_deps\mumps-src\src\tools_common.F.obj CMakeFiles\mumps_common.dir\_deps\mumps-src\src\mumps_pord.c.obj CMakeFiles\mumps_common.dir\_deps\mumps-src\src\mumps_thread.c.obj CMakeFiles\mumps_common.dir\_deps\mumps-src\src\mumps_save_restore_C.c.obj CMakeFiles\mumps_common.dir\_deps\mumps-src\src\mumps_config_file_C.c.obj CMakeFiles\mumps_common.dir\_deps\mumps-src\src\mumps_thread_affinity.c.obj CMakeFiles\mumps_common.dir\_deps\mumps-src\src\sol_common.F.obj CMakeFiles\mumps_common.dir\_deps\mumps-src\src\ana_blk.F.obj CMakeFiles\mumps_common.dir\_deps\mumps-src\src\mumps_register_thread.c.obj  -fuse-ld=llvm-lib -o mumps_common.lib /machine:x64    && cd ."
ifx: command line warning #10006: ignoring unknown option '/machine:x64'
ifx: error #10037: could not find 'C:\PROGRA~2\Intel\oneAPI\compiler\latest\windows\bin-llvm\llvm-lib'
[18/565] Building Fortran preprocessed CMakeFiles\cmumps.dir\_deps\mumps-src\src\cfac_distrib_distentry.F-pp.f
ninja: build stopped: subcommand failed.

**Comment by scivision** [comment]
It sounds like from your comment in the other thread that you did build successfully after this?

**Reply by 1lihua** [reply, marked as answer]
Yes, I switched to another computer and bulid mumps successfully. But there were some issues when I added mumps to Visual Studio. My folder \mumps-main\build\_deps\mumps-src\lib is empty. Whether I need to continue installing MUMPS in the folder_ 5.6.1in \mumps-main\build\_deps\mumps-src.
